### ISSUE TITLE
Check if wpautop is currently enabled and use it, if it is

### ISSUE THREAD

**Issue by rraub** [gios-asu/wp-front-end-editor#40]
We can call `has_filter` [codex](https://codex.wordpress.org/Function_Reference/has_filter) to see if wp_autop is being used and if it is we should run the content through it. 

This will make our fork of this plugin work for more people.

**Comment by rraub** [comment]
There is also some javascript we might need to change 
[Here is one change Ivan made](https://github.com/iseulde/wp-front-end-editor/commit/3c51e1ee0368c1ceebb4628615371f7bf236baa3)

**Comment by rraub** [comment]
Sorry I should have rephrased what I was asking for. The commit I referenced was a change Ivan made when he was disabling the wpautop, instead replacing it with some simple sanitization it looks like. What we are going to need to do in that javascript function call 

```
returnContent = window.switchEditors.pre_wpautop( returnContent );
```

 if wpautop is an active filter or call our quick and dirty alternative: 

```
returnContent = returnContent.replace(/<p>\[/g, '[');
returnContent = returnContent.replace(/\]<\/p>/g, ']');
returnContent = returnContent.replace(/<br \/>/g, '');
returnContent = returnContent.replace(/<p>&nbsp;<\/p>/g, '<br />');
```

Also it looks like the wpautop stuff got re-enabled in our last sync that we did with the source repo: https://github.com/gios-asu/wp-front-end-editor/commit/be814866d7fc86751f3b01248692425eefcdaf22

**Comment by chasethenag420** [comment]
Implemented it please review

**Comment by rraub** [comment]
closed in #42
